detailed_docs/TRS/RC/GetInteriorVehicleDataConsent.md

```diff
@@ -2,23 +2,61 @@
 
 1.
 In case
-- SDL received OnRemoteControlSettings notification from HMI with "ASK_DRIVER" access mode
-- and RC application (in HMILevel FULL) requested access to remote control module that is already allocated to another RC application
-- and SDL requested user consent from HMI via GetInteriorVehicleDataConsent
-- and user allowed access to RC module for requested application
+- SDL received OnRemoteControlSettings notification from HMI with `accessMode=ASK_DRIVER`
+- SDL received GetInteriorVehicleDataConsent request from mobile with valid `moduleIds`
+- SDL doesn't have internally cached concent value
 
 SDL must
-- allocate access to RC module to requested application
-- process control request from this application
-- respond with resultCode SUCCESS
-- send OnRCStatus to HMI with this RC module in `allocatedModules`
-- send OnRCStatus to the application with this RC module in `allocatedModules` and with allowed:true
+- resend GetInteriorVehicleDataConsent to HMI
+- resend GetInteriorVehicleDataConsent response from HMI to mobile 
+- save driver's decision to cache
 
 2.
 In case
-- SDL received OnRemoteControlSettings notification from HMI with "ASK_DRIVER" access mode
+- SDL received OnRemoteControlSettings notification from HMI with `accessMode=AUTO_ALLOW`
+- SDL received GetInteriorVehicleDataConsent request from mobile with valid `moduleIds`
+
+SDL must
+- respond with `allowed=true` for all `moduleIds` to the app
+- NOT resend GetInteriorVehicleDataConsent to HMI
```

Comment:
@KhrystynaDubovyk Please add next expectation into "SDL must" section according last changes in GetInteriorVehicleDataConsent implementation:
```
- does not save driver's decision to cache
```

Reply:
https://github.com/smartdevicelink/sdl_requirements/pull/129/commits/ba93fcee6437d4ccda780c5781d966c0769427f0